pkg/controllermanager/deployer/deployer.go

```diff
@@ -734,6 +734,17 @@ func (deployer *Deployer) handleBase(baseCopy *appsapi.Base) error {
 		return err
 	}
 
+	// If the sub associated with base is a dividing strategy, check if there are derived loc resources, if not, do not create desc
+	if sub, _ := deployer.subsController.FindSubByUID(baseCopy.Labels[known.ConfigSubscriptionUIDLabel]); sub != nil {
+		if sub.Spec.SchedulingStrategy == appsapi.DividingSchedulingStrategyType {
+			if locs, _ := deployer.locLister.Localizations(baseCopy.Namespace).List(labels.SelectorFromSet(labels.Set{
```

Comment:
We still need a way to ensure these localizations are up to date. cc @abstractmj for a look.

For sure, we can use the count `len(locs)` for new base objects.

Reply:
maybe we can associate the resourceversion of loc and sub

Reply:
> maybe we can associate the resourceversion of loc and sub

There are some cases that those updates are triggered by manifests/feeds.

I think we should add hash labels for these generated localizations (for replica diving case). These hash labels should indicate whether manifest/feed and finv are up to date.

Reply:
I agree with your idea, because the current logic is that manifests will be directly synchronized to description after updating the resource size, which is not reasonable.

By the way, in the case of dividing, can we just give up using localization for replicas settings? At the same time, you can consider putting the latest feedinventory version in the sub.

Reply:
The essential reason is the logic of controller-manager is independent of the logic of scheduler